Cargo.toml

```diff
@@ -46,3 +46,4 @@ base64 = "0.22.1"
 pico-args = "0.5"
 tiny-skia-path = "0.11.4"
 xmlwriter = "0.1"
+bencher = "0.1"
```

Comment:
This one isn't needed.

Reply:
Yes but it makes the IDE a lot calmer.

Reply:
Then it's an IDE bug. `ttf-parser/benches` has its own `Cargo.toml`. It's a completely separate entity.